src/main/webapp/app/exercise/review/review-comment-widget-manager.ts

```
@@ -0,0 +1,292 @@
+     */
+    renderWidgets(): void {
+        this.addSavedWidgets();
+        this.addDraftWidgets();
+    }
+
+    /**
+     * Pushes the latest submit availability to all draft widgets.
+     */
+    updateDraftInputs(): void {
+        const canSubmit = this.config.canSubmit();
+        this.draftWidgetRefs.forEach((ref) => ref.setInput('canSubmit', canSubmit));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)